### PR TITLE
Update to 2.13 family of things

### PR DESCRIPTION
**Pull request by cibernox**

### CODE REVIEW

File: jsconfig.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"compilerOptions":{"target":"es6","experimentalDecorators":true},"exclude":["node_modules","bower_components","tmp","vendor",".git","dist"]}
```
Comment (rwjblue) [inline jsconfig.json]: What is this file?

File: addon/index.js [hunk 2]
```diff
@@ -118,14 +118,13 @@ export default Ember.EventDispatcher.extend({
 
     let handleEvent = this._eventHandlers[event] = (event) => {
       let target = event.target;
-      let result;
 
       do {
         if (viewRegistry[target.id]) {
-          result = viewHandler(target, event);
```
Comment (rwjblue) [inline addon/index.js]: Do we just not need to track the return values?

File: jsconfig.json (hunk 1 above)
Comment (cibernox) [inline jsconfig.json]: Crap, that was my editor. I'll remove it.

File: addon/index.js (hunk 2 above)
Comment (cibernox) [inline addon/index.js]: ESLint properly warned me that we weren't doing anything with that variable.

Comment (rwjblue) [inline addon/index.js]: :thumbsup:

**Pull request merged**